tests/test_algorithm.py

```diff
@@ -4310,7 +4319,10 @@ def init_class_fixtures(cls):
         super(TestOrderAfterDelist, cls).init_class_fixtures()
         cls.data_portal = FakeDataPortal(cls.env)
 
-    def test_order_in_quiet_period(self):
+    @parameterized.expand([[1], [2]])
+    def test_order_in_quiet_period(self, sid):
```

Comment:
When using `expand`, it is useful to include a `name` parameter to show which condition is being tested.
e.g.:
```
@parameterized.expand([('auto_close_before_end_date', [1]), ('auto_close_after_end_date', [2])])
def test_order_in_quiet_period(self, name, sid):
    ...
```